Makefile

```diff
@@ -1,12 +1,12 @@
 # Versions from versions.txt
-VERSION ?= "$(shell grep -v '\#' versions.txt | grep operator= | awk -F= '{print $$2}')"
+VERSION ?= dev
```

Comment:
I assume this affects the actual image build flow for public

Reply:
ahh, yeah I'd prefer we leave this to what is in versions.txt ... this defines the default images used for the operator if one is not provided.